packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -81,8 +81,9 @@
           v-if="debug?.commitInfo?.authorName"
           data-cy="debug-header-author"
         >
-          <i-cy-general-user_x16
-            class="mr-1 mr-11px icon-dark-gray-500 icon-light-gray-100 icon-secondary-light-gray-200"
+          <UserAvatar
+            class="h-16px mr-7px w-16px"
```

Comment:
```suggestion
            class="h-16px mr-8px w-16px"
```
Any reason for this not being an increment of 4?